spark_log_parser/loaders/__init__.py

```diff
@@ -46,12 +47,20 @@ class FileChunkStreamWrapper:
     Note #2 - once `chunks` is consumed, calls to read/lines will return empty byte buffers
     """
 
-    def __init__(self, chunks: Iterator[bytes], maximum_file_size: int = None):
+    def __init__(self, chunks: Iterator[bytes], maximum_file_size: int = ArchiveExtractionThresholds.size):
```

Comment:
a data class does work for this! But instead of using Pydantic's `@dataclass`, I'm using the native one